internal/stores/sql.go

```diff
@@ -271,6 +274,7 @@ func (s *SQLStore) retryTransaction(fc func(tx *gorm.DB) error, opts ...*sql.TxO
 		if err == nil {
 			return nil
 		}
+		s.logger.Warn(context.Background(), fmt.Sprintf("transaction attempt %d/%d failed, err: %v", i+1, 5, err))
```

Comment:
Warn is the default log level so it will be visible by default. If it turns out it's too verbose we can switch it to trace.